src/libs/actions/Policy.js

```diff
@@ -154,6 +154,16 @@ function isAdminOfFreePolicy(policies) {
     return _.some(policies, (policy) => policy && policy.type === CONST.POLICY.TYPE.FREE && policy.role === CONST.POLICY.ROLE.ADMIN);
 }
 
+/**
+ * Is the user an admin of a corporate policy
+ *
+ * @param {Array} policies
+ * @returns {Boolean}
+ */
+function isAdminOfControlPolicy(policies) {
+    return _.some(policies, (policy) => policy && policy.type === CONST.POLICY.TYPE.CORPORATE && policy.role === CONST.POLICY.ROLE.ADMIN);
```

Comment:
We don't need to check for the policyID here? So if I'm an admin of any control policy this would return true?

Reply:
We supply the method with a single policy `return Policy.isAdminOfControlPolicy([policy])`, I reused code from free policies logic and I think it's not clear here, I'll create a new method `isAdminOfPolicy` which takes a `policyID`, sounds good? Maybe we already have something similar.

Reply:
Removed this method as it's not really needed for our case.